engine/Dockerfile

```
@@ -1,4 +1,10 @@
 FROM python:3.11.3-slim-buster AS base
+
+# Create a group and user to run an app
+ENV APP_USER=appuser
+RUN groupadd -r -g 1000 ${APP_USER} && \
+    useradd --no-log-init -r -u 1000 -g ${APP_USER} ${APP_USER}
```

same suggestion here. also, why the `--no-log-init` flag?
```suggestion
    useradd --no-log-init --system --uid 1000 --gid ${APP_USER} ${APP_USER}
```

---

I don't remember where I found the recommendation to use `--no-log-init`. With this flag `useradd` does not write to `/var/log/faillog` and `/var/log/lastlog` and saves around `300Kb` 😄 It's not a lot but I think it's not a bad flag for containerized user.